quinn-proto/src/connection/paths.rs

```diff
@@ -163,3 +170,453 @@ impl RttEstimator {
         }
     }
 }
+
+// Implements Datagram Packetization Layer Path Maximum Transmission Unit Discovery
+//
+// https://www.rfc-editor.org/rfc/rfc8899
+pub struct MtuDiscovery {
+    /// MtuDiscovery config
+    config: Option<PlpmtudConfig>,
+    /// Next instant at which mtud should be activated
+    next_mtud_activation: Option<Instant>,
+    /// Detected MTU for the path
+    current_mtu: u16,
+    /// The lower bound for the current binary search
+    search_lower_bound: u16,
+    /// The upper bound for the current binary search
+    search_upper_bound: u16,
+    /// The UDP payload size we are currently probing
+    probing_udp_payload_size: u16,
+    /// Packet number of an in-flight probe, if any
+    probe_packet_number: Option<u64>,
+    /// Lost probes at the current probe size
+    lost_probes: usize,
+    /// A count of the number of packets with a size > BASE_UDP_PAYLOAD_SIZE lost since
+    /// the last time a packet with size equal to the current MTU was acknowledged
+    black_hole_counter: u8,
+    /// The largest acked packet of size `max_udp_payload_size`
```

Comment:
Outdated comment? There's no field by that name in this struct.

Reply:
True, thanks!